tutorial/inputoutput.po

```diff
@@ -478,13 +477,14 @@ msgid ""
 "``f.read()`` will return an empty string (``''``).  ::"
 msgstr ""
 "Pour lire le contenu d'un fichier, appelez ``f.read(taille)`` : elle lit une "
```

Comment:
```suggestion
"Pour lire le contenu d'un fichier, appelez ``f.read(taille)`` : elle lit une "
```
/tmp/padpo_zivx6y2p/tutorial/inputoutput.po: 468 [     NBSP     ] There should be a non-breakable space before ":": between ###er, appelez « f.read(taille) » ### and ###: elle lit une certaine quantit###